.gitignore

```diff
@@ -35,3 +35,8 @@ out/
 
 productionStats/*.png
 productionStats/*.log
+
+# these files get easily created when running the tools from the IDE
```

Comment:
You are probably running the tools from the wrong working directory. It should always be the root of the repository not jsettlers.tools/
If you do it like this then these ignores are unnecessary

Reply:
Agree. I am just hitting the 'run' button in the IDE. It works quite well, despite the fact that the editor creates these nasty files. This is just a safety so we do not add them to version control.

Reply:
Netbeans does not execute the whole project, but I can select the subprojects and hit the run button there. In that case Netbeans will execute such as

```
cd <...>/settlers-installer/app; ../gradlew --configure-on-demand /.. run
```

While it does not look wrong, the current working directory is always on the subproject.
I failed reconfiguring Netbeans to invoke this differently. But this is outside the scope of enhancing the tools only, so I will remove it for now.